src/main/java/cl/gendigital/gendeporte/users/api/responses/user/get/GetUserResponse.java

```diff
@@ -0,0 +1,18 @@
+package cl.gendigital.gendeporte.users.api.responses.user.get;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Builder
+public class GetUserResponse {
+    private String username;
+    private String email;
+    private String validation_code;
+    private LocalDateTime created_at;
+    private LocalDateTime enabled_at;
+    private LocalDateTime updated_at;
```

Comment:
Los nombres de variables deben ser camelCase y no snake_case.

Reply:
Creo que esto fue para resolver que las respuestas json sean snake_case, sin embargo en java no son asi. Hay que usar una configuración del conversor java-json que tiene springboot, basado en la libreria Jackson. Se debe utilizar `@JsonProperty`. Puedes ver un ejemplo [aquí](https://www.concretepage.com/jackson-api/jackson-jsonproperty-and-jsonalias-example)